### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ktangsali**
# Modulus Pull Request

## Description
Updates Readme and removes reference to the Launch Repo.

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by ram-cherukuri — APPROVED** [review]
We can also hide the MTC link for now and add a section called domain specific packages to include Earth2MIP here.

File: README.md [hunk 1]
```diff
@@ -28,11 +28,20 @@ Detailed information on features and capabilities can be found in the [Modulus d
...
+  framework for building, training, and fine-tuning deep learning models using
+  state-of-the-art Physics-ML methods.
+- [Modulus Symbolic (Beta)](https://github.com/NVIDIA/modulus-sym): Framework
+  providing pythonic APIs, algorithms and utilities to be used with Modulus
+  core to physics inform model training as well as higher level abstraction
+  for domain experts.
+
+### Domain Specific Packages
+
+- [Earth-2 MIP (Alpha)](https://github.com/NVIDIA/earth2mip): Python framework
```
Comment (NickGeneva) [inline README.md:41]: Change this to Beta, I need to update the read me.

**Review by NickGeneva — APPROVED** [review]
lgtm, only one minor comment.

File: README.md (hunk 1 above)
Comment (ktangsali) [reply README.md]: Done!

**Pull request merged**